File: HashTable.java
import java.util.HashMap;

public class HashTable {
    public static void main(String[] args) {
        HashMap<String, Integer> hashTable = new HashMap<>();

        hashTable.put("apple", 3);
        hashTable.put("banana", 5);
        hashTable.put("orange", 2);

        // Выбираем значение
        System.out.println("Number of apples: " + hashTable.get("apple"));

        // Проверяем существование ключа
        if (hashTable.containsKey("banana")) {
            System.out.println("Bananas count: " + hashTable.get("banana"));
        }

        // Удаляем ключ и значение
        hashTable.remove("orange");
        System.out.println("Updated hash table: " + hashTable);
    }
}

File: hashtable.py
hash_table = {}

hash_table["apple"] = 3
hash_table["banana"] = 5
hash_table["orange"] = 2

# Выбираем значение
print("Number of apples:", hash_table["apple"])

# Проверяем существование ключа
if "banana" in hash_table:
    print("Bananas count:", hash_table["banana"])

# Удаляем ключ и значение
del hash_table["orange"]
print("Updated hash table:", hash_table)
